content/spin/kv-store-tutorial.md

```diff
@@ -184,7 +168,7 @@ command = "tinygo build -target=wasi -gc=leaking -no-debug -o main.wasm main.go"
 
 ## Using the Spin SDK
 
-In this section, we use the Spin SDK to open and persist our application's data inside our default key/value store. This is a special store that every environment running Spin applications will make available for their application. As mentioned above, the store is essentially an embedding of [SQLite](https://www.sqlite.org/index.html) within the Spin framework.
+In this section, we use the Spin SDK to open and persist our application's data inside our default key/value store. This is a special store that every environment running Spin applications will make available for their application. You can choose between various store implementations by modifying [the runtime configuration](dynamic-configuration.md#key-value-store-runtime-configuration). The default implementation uses [SQLite](https://www.sqlite.org/index.html) within the Spin framework.
```

Comment:
The last two sentences should be moved off to an aside; and that aside should be in the section about the default store, not in the section about the Spin SDK.